pkg/spec/gitops.go

```diff
@@ -1048,7 +1048,17 @@ func parsePolicyInstallSoftware(baseDir string, teamName *string, policy *Policy
 		}
 		var policyInstallSoftwareSpec fleet.SoftwarePackageSpec
 		if err := YamlUnmarshal(fileBytes, &policyInstallSoftwareSpec); err != nil {
-			return MaybeParseTypeError(policy.InstallSoftware.PackagePath, []string{"policy", "install_software", "package_path"}, err)
+			// see if the issue is that a package path was passed in that references multiple packages
+			var multiplePackages []*fleet.SoftwarePackageSpec
+			if err := YamlUnmarshal(fileBytes, &multiplePackages); err != nil || len(multiplePackages) == 0 {
+				return fmt.Errorf("file %q does not contain a valid software package definition", policy.InstallSoftware.PackagePath)
+			}
+
+			if len(multiplePackages) > 1 {
+				return fmt.Errorf("file %q contains multiple packages, so cannot be used as a target for policy automation", policy.InstallSoftware.PackagePath)
+			}
+
+			policyInstallSoftwareSpec = *multiplePackages[0]
```

Comment:
No need for a slice of pointers here.

```suggestion
			var multiplePackages []fleet.SoftwarePackageSpec
			if err := YamlUnmarshal(fileBytes, &multiplePackages); err != nil || len(multiplePackages) == 0 {
				return fmt.Errorf("file %q does not contain a valid software package definition", policy.InstallSoftware.PackagePath)
			}

			if len(multiplePackages) > 1 {
				return fmt.Errorf("file %q contains multiple packages, so cannot be used as a target for policy automation", policy.InstallSoftware.PackagePath)
			}

			policyInstallSoftwareSpec = multiplePackages[0]
```